cav_msgs/msg/TrajectoryPlan.msg

```diff
@@ -0,0 +1,10 @@
+# TrajectoryPlan.msg
```

Comment:
I think we should have a header in any message which will be published on its own topic.

Reply:
It is very hard to match event occurrence without it.